# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1312864315 25200
#      Mon Aug 08 21:31:55 2011 -0700
# Node ID 9f2679a1436686171d1facd918acf83d0347c31d
# Parent  1eebae7ac5d2dcb4ad5d924ac9b87ed64ba18d8a
# Parent  70e9ffbc4c763574b60c8d145e8250e1966b5b84
maint: Periodic merge of stable to default.

diff --git a/scripts/sparse/svds.m b/scripts/sparse/svds.m
--- a/scripts/sparse/svds.m
+++ b/scripts/sparse/svds.m
@@ -246,18 +246,18 @@ endfunction
 %! k = 7;
 %! A = sparse([3:n,1:n,1:(n-2)],[1:(n-2),1:n,3:n],[ones(1,n-2),0.4*n*ones(1,n),ones(1,n-2)]);
 %! [u,s,v] = svd(full(A));
 %! s = diag(s);
 %! [~, idx] = sort(abs(s));
 %! s = s(idx);
 %! u = u(:,idx);
 %! v = v(:,idx);
-%! randn ('state', 35);      % Initialize to make normest function reproducible
-%! rand ('state', 35)
+%! randn ('state', 42);      % Initialize to make normest function reproducible
+%! rand ('state', 42)
 %! opts.v0 = rand (2*n,1); % Initialize eigs ARPACK starting vector
 %!                         % to guarantee reproducible results
 %!test
 %! [u2,s2,v2,flag] = svds(A,k);
 %! s2 = diag(s2);
 %! assert(flag,!1);
 %! assert(s2, s(end:-1:end-k+1), 1e-10);
 %!testif HAVE_UMFPACK
